pages.pt_BR/osx/caffeinate.md

```diff
@@ -0,0 +1,16 @@
+# caffeinate
+
+> Evita que o macOS entre em suspensão (repouso).
+> Mais informações: <https://ss64.com/osx/caffeinate.html>.
+
+- Evitar a suspensão por uma hora (3600 segundos):
```

Comment:
For Romance languages like es, or pt, I think "presente indicativo" fits better.